Specs/Scene/ModelExperimental/ModelExperimentalAnimationCollectionSpec.js

```diff
@@ -615,6 +616,100 @@ describe("Scene/ModelExperimental/ModelExperimentalAnimationCollection", functio
     });
   });
 
+  it("animates with an explicit animation time", function () {
+    return loadAndZoomToModelExperimental(
+      {
+        gltf: animatedTriangleUrl,
+      },
+      scene
+    ).then(function (model) {
+      const time = JulianDate.fromDate(
+        new Date("January 1, 2014 12:00:00 UTC")
+      );
+      const animationCollection = model.activeAnimations;
+      animationCollection.animateWhilePaused = false;
```

Comment:
You can remove this line for the `ModelExperimental` spec. The models get deleted after every test here (which is not true for `Model`) so after creation, this will already default to false.

Reply:
This made me realize that I should set animateWhilePaused to false in ModelSpec.js at the end of "animates while paused with an explicit animation time".

So I did, and it broke "animates with a multiplier". The reason being that the first render in "animates with a multiplier" uses the same scene time as all the renders in "animates while paused with an explicit animation time", so the first render relied on having animateWhilePaused set, and after fixing that it took the _previousTime shortcut. So I also changed the render time for "animates while paused with an explicit animation time".

So yes, making each test start fresh is a much better design decision...